PWGCF/Flow/Tasks/CMakeLists.txt

```diff
@@ -58,3 +58,8 @@ o2physics_add_dpl_workflow(flow-sp
                     SOURCES flowSP.cxx
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::GFWCore
                     COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(flow-pbpb-reso
```

Comment:
There is nothing in the analysis really linked to PbPb so please, remove PbPb from the task executable name and preferably use `resonances-gfw-flow` if you are going to rely on GFW or `resonances-flow` if not as task name. Modify accordingly the source file name and the task `struct` name